README.md

```diff
@@ -1,218 +1,80 @@
+```java
 @SpringBootApplication
-public class DemoApplication {
-
-    public static void main(String[] args) {
-        SpringApplication.run(DemoApplication.class, args);
-    }
-
-    @Value("${cloudant.account}")
-    private String account;
-
-    @Value("${cloudant.username}")
-    private String username;
-
-    @Value("${cloudant.password}")
-    private String password
-
-    @Bean
-    public ClientBuilder builder() {
-        ClientBuilder builder = ClientBuilder
-            .account(this.account)
-            .username(this.username)
-            .password(this.password);
-        return builder;
-    }
-}
-~~~
-
-application.properties:
-
-~~~
-cloudant.account=myAccount
-cloudant.username=myUsername
-cloudant.password=myPassword
-~~~
-
-### Spring Framework Applications
-
-You must provide the following properties to define the connection to your Cloudant instance:
-
-* `cloudant.url`
-* `cloudant.username`
-* `cloudant.password`
-
-To enable the creation of the `com.cloudant.client.api.CloudantClient` bean you must add an `com.cloudant.spring.framework.EnableCloudant` annotation to your application configuration:
+@RestController
+public class CloudantJavaSdkSpringSampleApplication {
+    
+	@Autowired
+	private Cloudant cloudant;
 
-~~~ java
-@Configuration
-@EnableWebMvc
-@EnableCloudant
-@ComponentScan
-public class SpringConfig {}
-~~~
+	public static void main(String[] args) {
+		SpringApplication.run(CloudantJavaSdkSpringSampleApplication.class, args);
+	}
 
-~~~ java
-@Autowired
-private CloudantClient client;
+	@GetMapping("/all_dbs")
+	public String allDbs() {
+		return cloudant.getAllDbs().execute().getResult().stream().collect(Collectors.joining("\n"));
+	}
 
-public List<String> getAllDbs() {
-    return client.getAllDbs();
 }
-~~~
-
-## Related documentation
-* [Cloudant library for Java](https://github.com/cloudant/java-cloudant/)
-* [Cloudant documentation](https://console.bluemix.net/docs/services/Cloudant/cloudant.html#overview)
-* [Cloudant Learning Center](https://developer.ibm.com/clouddataservices/cloudant-learning-center/)
-* [Spring Boot documentation](https://projects.spring.io/spring-boot/)
-* [Spring Framework documentation](https://projects.spring.io/spring-framework/)
-
-# Development
-
-For information about contributing, building, and running tests see the [CONTRIBUTING.md](CONTRIBUTING.md).
+```
 
-### Using in Other Projects
+Run application:
 
-The preferred approach for using cloudant-spring in other projects is to use the Gradle or Maven dependency as described above.
+```shell
+./gradlew bootRun
+```
 
-### License
+REST request:
+```shell
+tomblench:cloudant-java-sdk-spring-sample$ curl http://localhost:8080/all_dbs
+kafka-test-db-04178485-8f15-4c3b-9138-d6ed68edb1d4
+kafka-test-db-07b6c52a-e2e9-4da5-b641-6233367f8af9
+kafka-test-db-0810edcf-fff6-4093-af01-f081fda77e74
+kafka-test-db-08d3cef8-50db-42a5-af9f-991493cbce50
+kafka-test-db-0c8255b0-ee10-4d4c-b104-def796305ebe
+...
```

Comment:
Let's clean this up slightly (I think `curl` is a copy/paste error) and kafka is just a little confusing in this context
```suggestion
test-db-04178485-8f15-4c3b-9138-d6ed68edb1d4
test-db-07b6c52a-e2e9-4da5-b641-6233367f8af9
test-db-0810edcf-fff6-4093-af01-f081fda77e74
test-db-08d3cef8-50db-42a5-af9f-991493cbce50
test-db-0c8255b0-ee10-4d4c-b104-def796305ebe
...
```